.circleci/config.yml

```diff
@@ -99,6 +99,7 @@ jobs:
       ANSIBLE_HOST_KEY_CHECKING: False
     steps:
       - checkout
+      - *setup_secrets
```

Comment:
I don't see any changes in the steps below, why do you need that? Secrets are only needed and used explicitly here: https://github.com/aeternity/infrastructure/pull/264/files#diff-1d37e48f9ceff6d8030570cd36286a61R145

Reply:
it was required when there was playbook idempodency test, since we do not need them removed.